### DISCUSSION TITLE
Self hosted instance missing features

### DISCUSSION THREAD

**Discussion by franferri** [plausible/analytics#909] [Self-Hosted Support] [answered]
Totally new to the system.

2 missing features in the self hosted system.

**First:**
I'm trying to have the dashboard embedded, following [https://plausible.io/docs/embed-dashboard](https://plausible.io/docs/embed-dashboard) instructions, it states:

_2. Generate the embed code#
In the "Visibility" area of your website settings there's the "Embed dashboard" section where you can generate the embed code._

But it seems in the self hosted system there is no such section. Is this normal?

**Second**

In the live demo [https://plausible.io/plausible.io](https://plausible.io/plausible.io) in the "Top pages" block of the dashboard there are this captions: **Top Pages**, **Entry Pages**, **Exit Pages**

But in the self hosted version I don't see the last 2.

Is this missing or there is a reason for this? maybe I'm missing to configure something?

**Comment by iSCJT** [comment, marked as answer]
The self hosted version is only updated periodically once new features have been battle tested in the hosted version and bugs worked out. Some of the latest features aren't currently in the self hosted repo.

I believe the next self hosted update will be out in the summer.

**Reply by metmarkosaric** [reply]
Exactly as @iSCJT said! 

You can find more details on the schedule and the differences between cloud and self-hosting here: https://github.com/plausible/analytics/

And there's a beta of the next self-hosted release that we're looking for feedback for: https://github.com/plausible/analytics/discussions/901
